broadcast.go

```diff
@@ -2,39 +2,12 @@ package ibctest
 
 import (
 	"context"
-
-	"github.com/cosmos/cosmos-sdk/client/tx"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/strangelove-ventures/ibctest/broadcast"
+	"fmt"
 )
 
-// BroadcastTx uses the provided Broadcaster to broadcast all the provided messages which will be signed
-// by the User provided. The sdk.TxResponse and an error are returned.
-func BroadcastTx(ctx context.Context, broadcaster broadcast.Broadcaster, broadcastingUser broadcast.User, msgs ...sdk.Msg) (sdk.TxResponse, error) {
-	for _, msg := range msgs {
-		if err := msg.ValidateBasic(); err != nil {
-			return sdk.TxResponse{}, err
-		}
-	}
-
-	f, err := broadcaster.GetFactory(ctx, broadcastingUser)
-	if err != nil {
-		return sdk.TxResponse{}, err
-	}
-
-	cc, err := broadcaster.GetClientContext(ctx, broadcastingUser)
-	if err != nil {
-		return sdk.TxResponse{}, err
-	}
-
-	if err := tx.BroadcastTx(cc, f, msgs...); err != nil {
-		return sdk.TxResponse{}, err
-	}
-
-	txBytes, err := broadcaster.GetTxResponseBytes(ctx, broadcastingUser)
-	if err != nil {
-		return sdk.TxResponse{}, err
-	}
-
-	return broadcaster.UnmarshalTxResponseBytes(ctx, txBytes)
+// BroadcastTx is deprecated.
+//
+// Deprecated: Use chain/cosmos.BroadcastTx for broadcasting to chains built with Cosmos SDK.
+func BroadcastTx(ctx context.Context, broadcaster any, broadcastingUser any, msgs ...any) (any, error) {
+	panic(fmt.Errorf("BroadcastTx is deprecated; use chain/cosmos.BroadcastTx"))
```

Comment:
If something is deprecated I would expect it to function as-is until it is removed at a later date. In this case maybe it's fine to just remove this function altogether rather than mark as deprecated. WDYT?

Reply:
I opted for a panic here to warn the consumer where the new functionality lives because it may not be obvious. In this case, the consumer also gets a compile time error because of the `any` refactors, specifically with the return value. But given I think ibc-go is the main consumer, and they are aware, I'll simply remove it now. 

@chatton Our current ibctest attitude is the cliche of "move fast and break things." We are currently not optimizing for backwards compatability. The library consumer is expected to pin to a specific git sha. When we move to a more traditional sematic versioning, we'll optimize more for backwards compat. At this time, I don't have a timeline for moving to semver.